src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -144,6 +199,15 @@
   box-shadow: var(--pf-c-context-selector__menu--BoxShadow);
 }
 
+.pf-c-context-selector__menu-item-icon {
```

Comment:
Is this element used anywhere?

Reply:
If this isn't being used (looks like it isn't) can you remove this block and any associated vars?